pkg/controller/dnszone/dnszone_controller_test.go

```diff
@@ -361,7 +361,7 @@ func TestReconcileDNSProviderForAzure(t *testing.T) {
 		},
 		{
 			name:    "Create Managed Zone, No ZoneName Set",
```

Comment:
No need to mention the "No ZoneName set" part. The DNSZone does not contain such a field.
```suggestion
			name:    "Create Managed Zone",
```